src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,48 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Film = {
+  [key: string]: string;
+};
```

Comment:
```suggestion
type Props = {
  film: Film;
};
```
You should describe props here